packages/core/src/networks.ts

```diff
@@ -337,44 +337,3 @@ export const shouldUseHigherMaxGasLimit = (chainId: bigint) => {
     throw new Error(`Unsupported network id ${chainId}`)
   }
 }
-
-/**
- * The number of blocks that Hardhat rewinds when forking the given network. Rewinding the block
- * number protects against chain reorgs. Copied from Hardhat:
- * https://github.com/NomicFoundation/hardhat/blob/caa504fe0e53c183578f42d66f4740b8ec147051/packages/hardhat-core/src/internal/hardhat-network/provider/utils/reorgs-protection.ts
- */
-export const getLargestPossibleReorg = (chainIdStr: string): bigint => {
```

Comment:
I removed this function, which was copied from Hardhat, and instead import the identical function directly from Hardhat. We previously defined it ourselves because I was paranoid about the fact that this function exists in an `internal/` directory in their package. I was concerned that the location could change unpredictably. However, since we hardcode the `hardhat` version and are planning on continuing to do this when we implement the Sphinx binary, I don't think this is a concern. Importing it ensures that we use the correct reorg size, which is necessary to enable caching. (More context to come)